src/Microsoft.OpenApi.Hidi/OpenApiService.cs

```diff
@@ -304,37 +307,52 @@ private static Stream ApplyFilter(string csdl, string entitySetOrSingleton, XslC
             }
         }
 
+        public static IConfiguration GetConfiguration(string settingsFile)
```

Comment:
switch to private or internal and use friendly assemblies for unit tests. We don't want people to take a dependency on that.